lib/Screens/account_dashboard.dart

```diff
@@ -20,97 +25,453 @@ class _AccountDashboardState extends State<AccountDashboard> {
+            ),
+          ),
+        );
+      },
+    );
+  }
+
+  Widget _appBar() {
+    return Row(
+      children: <Widget>[
+        const CircleAvatar(),
+        const SizedBox(width: 15),
+        TitleText(
+          text: account.alias,
+        ),
+        const Expanded(
+          child: SizedBox(),
+        ),
+        GestureDetector(
+          onTap: () {
+            _showAccountChoosingBottomSheet();
+          },
+          child: Icon(
+            Icons.short_text,
+            color: Theme.of(context).iconTheme.color,
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _operationsWidget() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceAround,
+      children: <Widget>[
+        _icon(
+          Icons.transfer_within_a_station,
+          'Transfer',
+        ),
+        _icon(
+          Icons.delete_sweep,
+          'Unlink',
+        ),
+        _icon(
+          Icons.code,
+          'Qr Pay',
+        ),
+      ],
+    );
+  }
+
+  Widget _icon(IconData icon, String text) {
+    return Column(
+      children: <Widget>[
+        GestureDetector(
+          onTap: () {},
+          child: Container(
+            height: 80,
+            width: 80,
+            margin: const EdgeInsets.symmetric(
+              vertical: 10,
+            ),
+            decoration: const BoxDecoration(
+              color: Colors.white,
+              borderRadius: BorderRadius.all(
+                Radius.circular(
+                  20,
+                ),
+              ),
+              boxShadow: [
+                BoxShadow(
+                  color: Color(0xfff3f3f3),
+                  offset: Offset(
+                    5,
+                    5,
+                  ),
+                  blurRadius: 10,
+                )
+              ],
+            ),
+            child: Icon(icon),
+          ),
+        ),
+        Text(
+          text,
+          style: GoogleFonts.muli(
+            fontSize: 15,
+            fontWeight: FontWeight.w600,
+            color: const Color(
+              0xff76797e,
+            ),
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _transactionList() {
+    final List<Transaction> debitTransactionList =
+        getDebitTransactions(account.accountNumber);
+    return Column(
+      children:
+          debitTransactionList.map((Transaction t) => _transaction(t)).toList(),
+    );
+  }
+
+  void _showTransactionDetailBottomSheet(Transaction t) {
+    final IconData icon = t.status == Status.SUCCESSFUL
+        ? Icons.check_circle_outline
+        : (t.status == Status.PENDING ? Icons.assignment_late : Icons.cancel);
+    final Color textColor = t.status == Status.SUCCESSFUL
+        ? Colors.green
+        : (t.status == Status.PENDING ? Colors.yellow : Colors.red);
+    final String text = t.status == Status.SUCCESSFUL
+        ? 'Successful'
+        : (t.status == Status.PENDING ? 'Pending' : 'Error');
```

Comment:
Can you please move this under `lib/widgets` as well?

I think `AccountDashboard` shouldn't know about the logic details like choosing the appropriate colours etc. It would be better to keep the screen file simple.